node/src/BaseClient.ts

```diff
@@ -2818,6 +2819,46 @@ export class BaseClient {
         return this.createWritePromise(scriptInvocation);
     }
 
+    /**
+     * Returns stream entries matching a given range of IDs.
+     *
+     * See https://valkey.io/commands/xrange for more details.
+     *
+     * @param key - The key of the stream.
+     * @param start - The starting stream ID bound for the range.
```

Comment:
I think it is better to say "entry ID" or "stream entry ID" instead of "stream ID".

Reply:
Please update in other places or align with other API. Consider updating in other clients too if needed (in another PR).